resources/provider/provider.go

```diff
@@ -82,157 +82,158 @@ func Provider() *provider.Provider {
-			"rds.event_subscriptions":               rds.RdsEventSubscriptions(),
-			"rds.instances":                         rds.RdsInstances(),
-			"redshift.event_subscriptions":          redshift.EventSubscriptions(),
-			"redshift.clusters":                     redshift.RedshiftClusters(),
-			"redshift.subnet_groups":                redshift.RedshiftSubnetGroups(),
-			"route53.domains":                       route53.Route53Domains(),
-			"route53.health_checks":                 route53.Route53HealthChecks(),
-			"route53.hosted_zones":                  route53.Route53HostedZones(),
-			"route53.reusable_delegation_sets":      route53.Route53ReusableDelegationSets(),
-			"route53.traffic_policies":              route53.Route53TrafficPolicies(),
-			"s3.accounts":                           s3.S3Accounts(),
-			"s3.buckets":                            s3.S3Buckets(),
-			"sagemaker.endpoint_configurations":     sagemaker.SagemakerEndpointConfigurations(),
-			"sagemaker.models":                      sagemaker.SagemakerModels(),
-			"sagemaker.notebook_instances":          sagemaker.SagemakerNotebookInstances(),
-			"sagemaker.training_jobs":               sagemaker.SagemakerTrainingJobs(),
-			"secretsmanager.secrets":                secretsmanager.SecretsmanagerSecrets(),
-			"sns.subscriptions":                     sns.SnsSubscriptions(),
-			"sns.topics":                            sns.SnsTopics(),
-			"sqs.queues":                            sqs.SQSQueues(),
-			"ssm.documents":                         ssm.SsmDocuments(),
-			"ssm.instances":                         ssm.SsmInstances(),
-			"waf.rule_groups":                       waf.WafRuleGroups(),
-			"waf.rules":                             waf.WafRules(),
-			"waf.subscribed_rule_groups":            waf.WafSubscribedRuleGroups(),
-			"waf.web_acls":                          waf.WafWebAcls(),
-			"wafv2.ipsets":                          wafv2.Ipsets(),
-			"wafv2.managed_rule_groups":             wafv2.Wafv2ManagedRuleGroups(),
-			"wafv2.regex_pattern_sets":              wafv2.RegexPatternSets(),
-			"wafv2.rule_groups":                     wafv2.Wafv2RuleGroups(),
-			"wafv2.web_acls":                        wafv2.Wafv2WebAcls(),
-			"wafregional.rate_based_rules":          wafregional.RateBasedRules(),
-			"wafregional.rule_groups":               wafregional.RuleGroups(),
-			"wafregional.rules":                     wafregional.Rules(),
-			"wafregional.web_acls":                  wafregional.WebAcls(),
-			"workspaces.workspaces":                 workspaces.Workspaces(),
-			"workspaces.directories":                workspaces.Directories(),
-			"xray.encryption_config":                xray.EncryptionConfigs(),
-			//"iot.security_profiles": 				 iot.IotSecurityProfiles(), //TODO disabled because of api error NotFoundException: No method found matching route security-profiles for http method GET.
+			"accessanalyzer.analyzers":                 accessanalyzer.AccessAnalyzerAnalyzers(),
```

Comment:
Maybe we should add newline after some resource "class"es? wdyt?

Reply:
Good point, I'll do it at least for the `config`.